ceph.spec.in

```diff
@@ -81,24 +81,23 @@ BuildRequires:	python
 BuildRequires:	python-argparse
 BuildRequires:	python-nose
 BuildRequires:	python-requests
-%if ( 0%{?rhel} > 0 && 0%{?rhel} < 7 ) || ( 0%{?centos} > 0 && 0%{?centos} < 7 )
-BuildRequires:	python-sphinx10
-%endif
 BuildRequires:	python-virtualenv
 BuildRequires:	util-linux
 BuildRequires:	xfsprogs
 BuildRequires:	xfsprogs-devel
 BuildRequires:	xmlstarlet
 BuildRequires:	yasm
-%if 0%{?rhel} || 0%{?fedora} || 0%{?suse_version}
-BuildRequires:	snappy-devel
-%endif
 
 #################################################################################
 # specific
 #################################################################################
-%if ! 0%{?rhel} || 0%{?fedora}
+%if ! 0%{?rhel} || 0%{?fedora} || 0%{?suse_version}
```

Comment:
Yeah, you can just remove the `!` operator and use the line that @dachary wrote above. RHEL ships `sharutils` (and `snappy-devel`), so there's no need to specify "if-not-rhel" here.